### PR TITLE
Feature/router css results

### LINKED ISSUE #1: As a user, I can see a title on a header when the page loads.

### LINKED ISSUE #2: As a user, I can see directive text telling me how to use the app. This is under the header.

### LINKED ISSUE #8: As a user, I should see a welcoming image and a pleasing color palette.
As a user, I should see a welcoming image and a pleasing color palette. Consistent styles are used for buttons and text.

### LINKED ISSUE #15: Add router, view landing/search page at '/'

### LINKED ISSUE #17: Set zip code in state when form is submitted to it can be passed to results as a prop

### LINKED ISSUE #18: View a results page of all markets near your zip

### LINKED ISSUE #19: Results page should route to '/markets'

### LINKED ISSUE #21: style landing/search page

### PR DESCRIPTION
**Pull request by clairefields15**
### Acceptance Criteria
Added router to App, deleted Home component and moved to App so that we could route to '/markets' and pass props to Results. Added some minimal styling to Search/Home/Results page. Render Cards to Results page.

### Issues Resolved
closes #1 
closes #2 
closes #8 
closes #15 
closes #21 
closes #18 
closes #19 
closes #17 

### Code Reviewers
@alexmkio @Meekb @AshleyOh-bit

### CODE REVIEW

**Review by Meekb — APPROVED** [review]
Thanks for all the work on styling , format/refactoring, moving code to App and the collaborative review to run through all the changes.

File: src/utils/apiCalls.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const fetchGetResponse = async (endPoint: string) => {
-  let url = `https://search.ams.usda.gov/farmersmarkets/v1/data.svc/${endPoint}`
+export const fetchData = async (endPoint: string) => {
```
Comment (alexmkio) [inline src/utils/apiCalls.ts:1]: While we aren't doing POST or DELETE for this project my issue with this naming convention is that POST and DELETE would require separate functions and both technically Fetch data. Is there a reason to change the specificity here?

File: src/components/Results/Results.css [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+  gap: 1em;
+  margin: 1em;
+}
+
 /* --------- MEDIA QUERIES -----------  */
 
 @media screen and (min-width: 620px) {
+  .results-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (AshleyOh-bit) [inline src/components/Results/Results.css:13]: I think we can refactor these media queries - grid has a repeat/autofit/minmax feature, that will automatically update the number of cards per page based on the page size. This should DRY up our media queries significantly.

File: src/components/Search/Search.tsx [hunk 3]
```diff
@@ -1,45 +1,45 @@
+import { listeners } from 'process';
```
[the remark this answers is not in the document]
Comment (clairefields15) [reply src/components/Search/Search.tsx:1]: Umm that's a great idea, mustve been weird autocomplete

File: public/index.html [hunk 4]
```diff
@@ -15,6 +15,21 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fredericka+the+Great&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
```
[the remark this answers is not in the document]
Comment (clairefields15) [reply public/index.html:28]: Nice I'll do that

File: src/utils/apiCalls.ts (hunk 1 above)
Comment (clairefields15) [reply src/utils/apiCalls.ts:1]: Ahhh I see, that makes sense. I can change to getData ? I just thought it was overly wordy and confusing

File: src/components/Results/Results.css (hunk 2 above)
Comment (clairefields15) [reply src/components/Results/Results.css:13]: Oh sweet! I haven't used that before

**Pull request merged**